src/evm_transition_tool/besu.py

```diff
@@ -28,7 +28,7 @@ class BesuTransitionTool(TransitionTool):
 
     default_binary = Path("evm")
     detect_binary_pattern = compile(r"^Hyperledger Besu evm .*$")
-
+    t8n_use_stream = True
```

Comment:
We can remove this line for Besu, which doesn't use stream (file IO); it has a `t8n-server`. However, it overrides `evaluate()` to implement communication with its `t8n-server`. This means it doesn't make use of either `t8n_use_stream` or `t8n_use_server`; they can be left as the defaults. This flag is simply unused for Besu.

```suggestion
```

Reply:
I made a follow-up ticket to potentially clean-up these two t8n server evaluate methods #818, it's beyond the scope of this PR.